clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -425,4 +425,15 @@ DPCT_NON_ENUM_OPTION(
         "Query functionally compatible SYCL API to migrate CUDA API."),
     llvm::cl::value_desc("api"), llvm::cl::cat(DPCTCat), llvm::cl::Optional)
 
+DPCT_NON_ENUM_OPTION(
+    DPCT_OPT_TYPE(static llvm::cl::opt<bool>), UsePureSyclQueue,
+    "use-pure-sycl-queue",
```

Comment:
-helper-function-preference // The preference of helper function usage in migration.
=no-queue-device
Call SYCL API to get queue and device instead of calling helper function. Notice that the "
                   "runtime behavior may not be the same with the original "
                   "code because there is only one global definition of SYCL "
                   "queue and device, which will be passed to place where "
                   "SYCL queue and device are expected. Default: off

Reply:
fixed